### DISCUSSION TITLE
Printing Layout / Rotation?

### DISCUSSION THREAD

**Discussion by sousben** [python-escpos/python-escpos#473] [Ideas]
Hi is it possible to print several elements on the same line (e.g. text then QR code?)
Is it possible to rotate characters etc (I'm printing on 70x40mm labels but I would like to print on them as if they were 40x70)

Is my only option to create an image with a rendition of what I want to print, then rotate it and print it?

Thank you

**Comment by AustinZhang1024** [comment]
i think the only option is printing a image. now im using typst to generate a png file instead of pillow
